Sources/MockoloFramework/Utils/TypeParser.swift

```diff
@@ -685,14 +686,10 @@ public final class SwiftType {
          "UIColor": ".black",
          "UIFont": ".systemFont(ofSize: 12)",
          "UIImage": "UIImage()",
-         "UIView": "UIView(frame: .zero)",
-         "UIViewController": "UIViewController()",
-         "UICollectionView": "UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewLayout())",
-         "UICollectionViewLayout": "UICollectionViewLayout()",
-         "UIScrollView": "UIScrollView()",
```

Comment:
[question]

These changes aim to disable defaultValue of globalActor, but some users still use swift 5 mode, I think. so these changes might break backward compatibilities, in my understanding.
What do you think of it?

Reply:
This seems to be a breaking change, but imagine users who would be affected by this.
It shouldn't have been compiling correctly in the first place. That's what #171 is about.

The @MainActor attribute was introduced in Swift 5.1, and mockolo currently supports Swift 5.7 and above. There's no need to consider users with Swift versions below 5.1.

Reply:
I see. I misunderstood what this issue says.